CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Added map_v2_event_into_v1 method to Utils for all plugin classes to use. 
```

Comment:
I can add attribution for you post acceptance or you can do it prior, either way works.